hugo/assets/scss/base/base.scss

```diff
@@ -336,6 +336,7 @@ pre {
 }
 
 code {
+    display: inline-flex;
```

Comment:
this is now breaking the usage of code in some other places like https://deploy-preview-470--cue.netlify.app/examples/shortcodes/code-tabs/ and https://deploy-preview-470--cue.netlify.app/docs/howto/transform-json-with-cue/  where the columns are now not growing correctly anymore and the code is now displayed in a row because of the inline-flex (when you turn of this css line you can see the difference on this pages) . Can't we use inline-block? or maybe not change the display style but use white-space: pre or nowrap? We need to make sure the solution doesn't break other usages of the code tag.  

![Screenshot from 2024-09-19 15-03-46](https://github.com/user-attachments/assets/be5b896b-e099-4667-a0af-a6cfa5e94bfc)

Reply:
Replaced the display with white-space pre instead!